source/audio/Audio.cpp

```diff
@@ -303,6 +307,22 @@ void Audio::PlayMusic(const string &name)
 
 
 
+// Pause all active playback streams. Doesn't cause new streams to be paused.
```

Comment:
make it the same as in the header
```suggestion
// Pause all active playback streams. Doesn't cause new streams to be paused, and doesn't pause the music source.
```